packages/terra-action-header/src/terra-dev-site/doc/example/BorderlessActionHeader.jsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-alert */
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+// import ActionHeader from 'terra-action-header/lib/ActionHeader';
```

Comment:
This needs to be `import ActionHeader from 'terra-action-footer/lib/ActionHeader `. Our hot reloading can resolve this during testing, however we need this import to be correctly resolved for the terra-ui site.

Reply:
Not these only need to be updated in the example files.